db/migrate/20170501201633_add_canceled_to_order_detail.rb

```diff
@@ -0,0 +1,19 @@
+class AddCanceledToOrderDetail < ActiveRecord::Migration
+  def up
+    add_column :order_details, :canceled_at, :datetime
+    add_column :order_details, :canceled_by, :integer
+    add_column :order_details, :canceled_reason, :string
+
+    Reservation.find_each do |reservation|
+      reservation.order_detail.update_attributes(canceled_at: reservation.canceled_at,
+                                                 canceled_by: reservation.canceled_by,
+                                                 canceled_reason: reservation.canceled_reason)
```

Comment:
That's probably a smart idea. This also reminds me we'll need to make sure to inform NU's BI team that these columns will be moving. It'll probably be confusing if they keep pointing at the old columns, but no new data is getting populated into them.